topsort-api-v2.yml

```diff
@@ -293,6 +297,54 @@ components:
         - winners
         - error
 
+    # Dimensions:
+    #   type: object
+    #   description: The dimensions of this source.
+    #   properties:
+    #     width:
+    #       type: integer
+    #       format: int32
+    #       description: The source width in pixels.
+    #       minimum: 1
+    #     height:
+    #       type: integer
+    #       format: int32
+    #       description: The source height in pixels.
+    #       minimum: 1
+    #   required:
+    #     - width
+    #     - height
+
+    AssetSources:
+      description: The list of available sources for an asset (banner or video).
+      type: array
+      items:
+        type: object
```

Comment:
Ideally this should be its own type, otherwise generators will create an anonymous class, which makes it really awkward to use.

Reply:
I replaced the AssetSources with AssetSource.

Reply:
@pablo what name does it give to the Array type?

Reply:
`AssetSourcesInner`. It just creates the named based on the property name

Reply:
Ok, but the user of the API will have to guess that it got added the `Inner` suffix. Some other generators use different naming schemes.